playwright/src/main/java/com/microsoft/playwright/Accessibility.java

```diff
@@ -35,6 +46,28 @@ public SnapshotOptions withRoot(ElementHandle root) {
   default AccessibilityNode snapshot() {
     return snapshot(null);
   }
+  /**
+   * Captures the current state of the accessibility tree. The returned object represents the root accessible node of the page.
+   * <p>
+   * 
+   * <p>
+   * <strong>NOTE</strong> The Chromium accessibility tree contains nodes that go unused on most platforms and by
+   * <p>
+   * most screen readers. Playwright will discard them as well for an easier to process tree,
+   * <p>
+   * unless {@code interestingOnly} is set to {@code false}.
+   * <p>
+   * 
+   * <p>
+   * An example of dumping the entire accessibility tree:
+   * <p>
+   * 
+   * <p>
+   * An example of logging the focused node's name:
+   * <p>
+   * 
```

Comment:
What would be the plan here?

Reply:
Don't know yet, we'll have to figure out a story that would work across languages and be manageable in terms of long term maintenance. We brainstormed a few approaches like embedding language-specific snippets into api.md or approach documentation similar to localization strings but all of those had problems that we didn't like. For now I'm just removing the snippets to avoid confusion. I can try and remove the lines starting with 'An example of' too.